lib/IDE/ModuleInterfacePrinting.cpp

```diff
@@ -374,8 +374,10 @@ static bool printModuleInterfaceDecl(Decl *D,
 
 /// Sorts import declarations for display.
 static bool compareImports(ImportDecl *LHS, ImportDecl *RHS) {
-  auto LHSPath = LHS->getFullAccessPath();
-  auto RHSPath = RHS->getFullAccessPath();
+  // TODO: Probably buggy--thinks "import Foo" == "import Foo.Bar".
+  // ImportPathBase should provide universal comparison functions to avoid this.
```

Comment:
If you concur with this comment, I'll be happy to add an SR number. (I don't want to make functional changes in this PR if possible.)

Reply:
Seems like a bug to me too.

Reply:
https://bugs.swift.org/browse/SR-13490

I'll add the bug number to the comment so we don't lose track of it.

Reply:
I've added the bug number to the code.